lib/log-config.js

```diff
@@ -1,10 +1,14 @@
 var bunyan = require('bunyan');
+var PrettyStream = require('bunyan-prettystream');
```

Comment:
This library has the note "This library is only really meant for development and should not be used on production environments."

Do we have a really good reason to use this despite that note?